spec/controllers/club_applications_controller_spec.rb

```diff
@@ -19,7 +19,7 @@
         }.to change(ActionMailer::Base.deliveries, :size).by 2
 
         emails = ActionMailer::Base.deliveries.last(2)
-        my_email = 'team@hackedu.us'
+        my_email = 'team@hackclub.io'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.